lessons/6_objects_intro/homework.md

```diff
@@ -0,0 +1,23 @@
+# Lesson 6 Homework
+
+Start working on your little online store prototype and practice objects and arrays
+
+> Important: do not forget to do your homework on a separate branch with meaningful name
+
+1. Create a simple page to display products of your imaginary online store
+2. Decide on 1 type of items sold (ex. cars, dresses, goats)
+3. Create object representations of items you sell.
+    * There shall be 4 items sold
+    * Each item must have at least 4 simple attributes and 1 nested object
+    * ALL items should have exactly same structure
+4. Use array to store all objects together
+6. Display all your objects on a web-page
+
+# Deadline
+
+Saturday 18:00 15.04.2020
+
+## Lecture slides
+
+[Lesson 6 - intro to objects](https://mrtim.github.io/js-munich-2020-spring/lessons/6_objects_intro)
```

Comment:
How do we link the lecture slides from lesson 7 about arrays?

Reply:
I thought when slides are there we just replace it with a link and announce it on Thursday?

Reply:
Maybe put a placeholder that there would be another link in addition to that one? because homework would not change.

Reply:
but then link will be not working... Guess students might get confused.. not sure how to procede

Reply:
I think we should add the placeholder i.e Lesson 7 (coming in Thursday 23th )

Reply:
I have following text on  line 25: > Lecture about arrays will be available on 23.04.2020
Could you please suggest to what exactly I shall change it?